sed/io.py

```diff
@@ -18,10 +52,22 @@ def recursive_write_metadata(h5group, node):
             try:
                 h5group.create_dataset(key, data=str(item))
                 print("saved " + key + " as string")
-            except:
-                raise ValueError('Cannot save %s type'%type(item))
+            except BaseException:
```

Comment:
If there would be a BaseException, why raising a ValueError?

Reply:
Fair point, I replaced this now with a generic exception, as this is anyways an unkown error.